src/main/resources/assets/schemas/DataAccessRequest.yaml

```diff
@@ -22,6 +22,12 @@ properties:
   draft:
     type: boolean
     description: Draft status of this DAR
+  expired:
+    type: boolean
+    description: Expired status of this DAR
+  expiredAt:
+    type: number
+    description: When this DAR expires, in milliseconds since the epoch.  A value of -1 indicates this DAR does not have an expiration date because it is a draft.
```

Comment:
Another approach is to have these fields be `null` or missing unless `draft` is `false`. Or, instead of a sentinel value, use `Instant.MAX` which is a valid time offset and could be used in comparisons, vs `-1` which is invalid.

Reply:
OBE since expiredAt won't return when it's null.  Docs updated.